ceph/rados/core_workflows.py

```diff
@@ -986,6 +986,16 @@ def verify_reweight(self, affected_osds: list, osd_info: list) -> bool:
 
         return True
 
+    def list_pools_on_clustr(self) -> list:
+        """
+        Method fetches the list of pools from the cluster via ceph df command
+
+        Returns:
+            list of pools on the cluster
+        """
+        existing_pools = self.run_ceph_command(cmd="ceph df", client_exec=True)
+        return [ele["name"] for ele in existing_pools["pools"]]
```

Comment:
I believe `ceph osd pool ls -f json` would simplify the code
_**# ceph osd pool ls -f json
[".mgr","cephfs.cephfs.meta","cephfs.cephfs.data","pool_replica_less_min"]**_


`return self.run_ceph_command(cmd="ceph osd pool ls", client_exec=True)`

Let me know what you think

Reply:
Updated the code.